cmd/antrea-agent/options.go

```diff
@@ -127,8 +132,15 @@ func (o *Options) setDefaults() {
 	if o.config.ServiceCIDR == "" {
 		o.config.ServiceCIDR = defaultServiceCIDR
 	}
+	if o.config.PodTrafficEncapMode == "" {
+		o.config.PodTrafficEncapMode = types.PodEncapModeEncap.String()
+	}
+
 	if o.config.DefaultMTU == 0 {
-		if o.config.TunnelType == ovsconfig.VXLANTunnel {
+		ok, encapMode := types.GetPodEncapModeFromStr(o.config.PodTrafficEncapMode)
+		if ok && !encapMode.SupportsEncap() {
+			o.config.DefaultMTU = defaultMTU
```

Comment:
We should use defaultMTU only for encap, but not hybrid?

Reply:
supportEncap include encap and hybrid mode

Reply:
Got it.